server/controllers/orderControllers.js

```diff
@@ -0,0 +1,154 @@
+import Product from "../models/productModel.js";
+import Order from "../models/orderModel.js";
+
+export const createOrder = async (req, res) => {
```

Comment:
Good work. Also, allow users to be able to order directly from their cart.

Reply:
?? i dont understand this.

Reply:
anyways let me do the basics and move

Reply:
Ok, we will revisit in another PR.